custom_components/meural/pymeural.py

```diff
@@ -43,19 +51,31 @@ async def request(self, method, path, data=None) -> Dict:
             else:
                 kwargs["json"] = data
         with async_timeout.timeout(10):
-            resp = await self.session.request(
-                method,
-                url,
-                headers={
-                    "Authorization": f"Token {self.token}",
-                    "x-meural-api-version": "3",
-                },
-                raise_for_status=True,
-                **kwargs,
-            )
+            try:
+                resp = await self.session.request(
+                    method,
+                    url,
+                    headers={
+                        "Authorization": f"Token {self.token}",
+                        "x-meural-api-version": "3",
+                    },
+                    raise_for_status=True,
+                    **kwargs,
+                )
+            except ClientResponseError as err:
+                _LOGGER.warning('Meural: Sending Request failed. Re-Authenticating.')
+                self.token = None
+                return
```

Comment:
Don't ignore the warning but instead retry the request if it is not a new token
```suggestion
                # If a new token was just fetched and it fails again, just raise
                if fetched_new_token:
                    raise
                _LOGGER.warning('Meural: Sending Request failed. Re-Authenticating.')
                self.token = None
                return self.request(method, path, data)
```